README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
"letter x of y" block fails with output:
> Invalid operand type

<img width="228" alt="image" src="https://user-images.githubusercontent.com/126523/221427262-531219c7-c094-46d3-8078-92f4b3289fa7.png">

```xml
<xml xmlns="http://www.w3.org/1999/xhtml">
  <variables>
    <variable type="" id=";/fOvI=r,rhuvz3kg:!y" islocal="false" iscloud="false">i</variable>
    <variable type="" id="mXKmvt*Z7vJ|3:~+HU:#" islocal="false" iscloud="false">j</variable>
    <variable type="" id="2ogUd*aq6_gfGWacz#/2" islocal="false" iscloud="false">testvar</variable>
    <variable type="" id="q=qhREa7eU=jdV,%uUs-" islocal="false" iscloud="false">k</variable>
    <variable type="" id="m+BqD().ru``m17|I[)," islocal="false" iscloud="false">m</variable>
    <variable type="" id="#S;ca-yM_;tly+p4=K4o" islocal="false" iscloud="false">n</variable>
    <variable type="" id="zxEcNq=cqSYy+euCDP32" islocal="false" iscloud="false">o</variable>
  </variables>
  <block type="control_run" id="0fNAs-vY8^BL;[T$]eav" x="159" y="8">
    <next>
      <block type="data_setvariableto" id="4cVTC%HH)byt|f:g#n~c">
        <field name="VARIABLE" id="2ogUd*aq6_gfGWacz#/2" variabletype="">testvar</field>
        <value name="VALUE">
          <shadow type="text" id="%L,.yYJ%#uBN@-Mpj(98">
            <field name="TEXT">0</field>
          </shadow>
          <block type="operator_letter_of" id="AV*Z(ob+RcU+ZZ*LUl~I">
            <value name="LETTER">
              <shadow type="math_whole_number" id="%!sO9QkRua:yT1S-I.^M">
                <field name="NUM">1</field>
              </shadow>
            </value>
            <value name="STRING">
              <shadow type="text" id="3lPB[X8`PoL2s2XX4r0*">
                <field name="TEXT">world</field>
              </shadow>
            </value>
          </block>
        </value>
      </block>
    </next>
  </block>
</xml>
```